<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SampleDBWindow</class>
 <widget class="QDialog" name="SampleDBWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1516</width>
    <height>616</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>SampleDB</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QPushButton" name="removeSample_button">
       <property name="text">
        <string>Remove Sample</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QCheckBox" name="checkBox_4">
       <property name="text">
        <string>Shift vertically</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QCheckBox" name="normalizeBox">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Normalize</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QPushButton" name="editSample_button">
       <property name="text">
        <string>Edit Sample</string>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QPushButton" name="addSample_button">
       <property name="text">
        <string>Add Sample</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item alignment="Qt::AlignRight">
        <widget class="QLabel" name="label">
         <property name="maximumSize">
          <size>
           <width>300</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="text">
          <string>Skip data points:</string>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignLeft">
        <widget class="QLineEdit" name="skipdata">
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>10</string>
         </property>
         <property name="maxLength">
          <number>100</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="4">
      <widget class="QPushButton" name="exportPDF_button">
       <property name="text">
        <string>Export to  PDF</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <layout class="QGridLayout" name="gridLayout_3"/>
   </item>
   <item row="2" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
     <property name="centerButtons">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QTableWidget" name="SampleDBList">
     <property name="minimumSize">
      <size>
       <width>256</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>1677215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QAbstractScrollArea::AdjustToContents</enum>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="dragEnabled">
      <bool>false</bool>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="textElideMode">
      <enum>Qt::ElideRight</enum>
     </property>
     <property name="showGrid">
      <bool>true</bool>
     </property>
     <property name="gridStyle">
      <enum>Qt::SolidLine</enum>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <attribute name="horizontalHeaderCascadingSectionResizes">
      <bool>false</bool>
     </attribute>
     <row>
      <property name="text">
       <string>1</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>2</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>3</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>4</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>5</string>
      </property>
     </row>
     <column>
      <property name="text">
       <string>Sample ID</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Date</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>N</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Materials</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Period</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Magnetron Powers</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Bias</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Deposition time</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Background pressure</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Comments</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Include</string>
      </property>
     </column>
     <item row="0" column="0">
      <property name="text">
       <string>IFM20001</string>
      </property>
     </item>
     <item row="0" column="1">
      <property name="text">
       <string>2020-01-14</string>
      </property>
     </item>
     <item row="0" column="2">
      <property name="text">
       <string>100</string>
      </property>
     </item>
     <item row="0" column="3">
      <property name="text">
       <string>Ni/Ti</string>
      </property>
     </item>
     <item row="0" column="7">
      <property name="text">
       <string>Ni 5s, Ti 10s</string>
      </property>
     </item>
     <item row="0" column="9">
      <property name="text">
       <string>Bla bla bla</string>
      </property>
     </item>
     <item row="0" column="10">
      <property name="text">
       <string/>
      </property>
      <property name="checkState">
       <enum>Checked</enum>
      </property>
     </item>
     <item row="1" column="0">
      <property name="text">
       <string>IFM20002</string>
      </property>
     </item>
     <item row="1" column="1">
      <property name="text">
       <string>2020-01-14</string>
      </property>
     </item>
     <item row="1" column="2">
      <property name="text">
       <string>57</string>
      </property>
     </item>
     <item row="1" column="3">
      <property name="text">
       <string>Co/Yi</string>
      </property>
     </item>
     <item row="1" column="7">
      <property name="text">
       <string>Co 5s, Yi, 20s</string>
      </property>
     </item>
     <item row="1" column="9">
      <property name="text">
       <string>Test</string>
      </property>
     </item>
     <item row="1" column="10">
      <property name="text">
       <string/>
      </property>
      <property name="checkState">
       <enum>Unchecked</enum>
      </property>
     </item>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SampleDBWindow</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SampleDBWindow</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
